README.md

```diff
@@ -66,38 +67,10 @@ This project was started by [ashleygwilliams] and is co-maintained by [ashleygwi
 
 ## ⚡ Quickstart Guide
 
-1. Write a crate in Rust.
-2. Add `wasm-bindgen` to your `Cargo.toml`:
-
-  ```toml
-  [lib]
-  crate-type = ["cdylib"]
-
-  [dependencies]
-  wasm-bindgen = "0.2"
-  ```
-3. Add this to the top of your `src/lib.rs`:
-
-  ```rust
-  use wasm_bindgen::prelude::*;
-  ```
-
-4. Annotate your public functions with `#[wasm_bindgen]`, for example:
-
-  ```rust
-  #[wasm_bindgen]
-  extern {
-      pub fn alert(s: &str);
-  }
-
-  #[wasm_bindgen]
-  pub fn greet(name: &str) {
-      alert(&format!("Hello, {}!", name));
-  }
-  ```
-
-5. Install this tool: `cargo install wasm-pack`
-6. Run `wasm-pack build`, optionally, pass a path to a dir or a scope (see above for details)
-7. This tool generates files in a `pkg` dir
-8. To publish to npm, run `wasm-pack publish`. You may need to login to the
+1. Install this tool: `cargo install wasm-pack`
+1. Run `wasm-pack generate`.
+1. `cd hello-wasm`
+1. Run `wasm-pack build`, optionally, pass a path to a dir or a scope (see above for details)
```

Comment:
Since this is the "quick start" section, perhaps the `, optionally, ...` could be removed and/or just link to the reference?